task/rpm-ostree/0.2/MIGRATION.md

```diff
@@ -7,15 +7,27 @@ artifact, e.g. using the git-clone Task version 0.3 or newer.
 ## Action from users
 
 Update files in Pull-Request created by RHTAP bot:
-- Search for the task named `rpm-ostree`
-- if not maintaining a hermetic build provide the `SOURCE_ARTIFACT` parameter
-  value from the result of the  `git-clone` Task. For example:
+- if your pipeline includes a `prefetch-dependencies` task, as per default, add
+  the `SOURCE_ARTIFACT` and `CACHI2_ARTIFACT` from the results of the
+  `prefetch-dependencies` Task
 
 ```diff
  - name: rpm-ostree
    params:
 +  - name: SOURCE_ARTIFACT
-+    value: $(tasks.clone-repository.results.SOURCE_ARTIFACT) # clone-repository is the name of the git-clone Task in the Pipeline
++    value: $(tasks.prefetch-dependencies.results.SOURCE_ARTIFACT)
++  - name: CACHI2_ARTIFACT
++    value: $(tasks.prefetch-dependencies.results.CACHI2_ARTIFACT)
 ```
```

Comment:
The rpm-ostree task doesn't actually accept a CACHI2_ARTIFACT param

There's no dedicated pipeline that would use rpm-ostree as the build task (weird), but when I look at https://github.com/CentOS/centos-bootc/blob/main/.tekton/ostree-build.yaml, it doesn't use the prefetch task.

TLDR we should probably keep only the case where the pipeline doesn't include prefetch-dependencies